sdk/lib/gcc-compat/__throw_out_of_range_fmt.cpp

```diff
@@ -0,0 +1,42 @@
+/*
+ * PROJECT:     GCC c++ support library
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     __throw_out_of_range_fmt implementation
+ * COPYRIGHT:   Copyright 202024 Timo Kreuzer <timo.kreuzer@reactos.org>
```

Comment:
Wow it's far ahead in the future! (And same observation for the other files as well)